src/main/java/org/candlepin/subscriptions/security/SecurityConfig.java

```diff
@@ -48,6 +50,9 @@
     @Autowired
     private ObjectMapper mapper;
 
+    @Autowired
+    private ApplicationProperties appProps;
```

Comment:
SonarQube didn't like the injection here (since it is only used in one place), but given that #59 uses it too in a different place, I marked as WONTFIX. So don't bother changing. :smile:

Reply:
@SuppressWarnings("squid:S3305") on the field silences this issue.